basicdataset/src/test/resources/mlrepo/dataset/nlp/ai/djl/basicdataset/penntreebank/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+  "metadataVersion": "0.2",
+  "resourceType": "dataset",
+  "application": "nlp",
+  "groupId": "ai.djl.basicdataset",
+  "artifactId": "penntreebank",
+  "name": "penntreebank",
+  "description": "The Penn Treebank (PTB) project selected 2,499 stories from a three year Wall Street Journal (WSJ) collection of 98,732 stories for syntactic annotation.",
+  "website": "https://blog.salesforceairesearch.com/the-wikitext-long-term-dependency-language-modeling-dataset/",
+  "licenses": {
+    "license": {
+      "name": "LDC User Agreement for Non-Members",
+      "url": "https://catalog.ldc.upenn.edu/license/ldc-non-members-agreement.pdf"
+    }
+  },
+  "artifacts": [
+    {
+      "version": "1.0",
+      "snapshot": false,
+      "name": "penntreebank",
+      "files": {
+        "train":{
+          "uri" : "https://raw.githubusercontent.com/wojzaremba/lstm/master/data/ptb.train.txt",
```

Comment:
Is this right? I looked at the data, but it seems like just pre-processed text. I was expecting to see some kind of labels like https://www.ling.upenn.edu/courses/Fall_2003/ling001/penn_treebank_pos.html on all of the words

Reply:
Yes, this issue is not for part of speech tagging dataset. I'd already mentioned about this pre-processed dataset in issue #46, we are also working on adding a new POS dataset for issue #46. But in this new issue I just want to add this pre-processed dataset as WikiText2 which is just text data for language modeling.